src/SFA.DAS.Forecasting.Data/Configurations/AccountProjectionConfiguration.cs

```diff
@@ -23,6 +23,7 @@ public AccountProjectionConfiguration(string schema)
             Property(x => x.Year).HasColumnName(@"Year").HasColumnType("int").IsRequired();
             Property(x => x.FundsIn).HasColumnName(@"FundsIn").HasColumnType("decimal").IsRequired().HasPrecision(18,2);
             Property(x => x.TotalCostOfTraining).HasColumnName(@"TotalCostOfTraining").HasColumnType("decimal").IsRequired().HasPrecision(18,2);
+            Property(x => x.TransferOutTotalCostOfTraining).HasColumnName(@"TransferOutTotalCostOfTraining").HasColumnType("decimal").IsRequired().HasPrecision(18, 2);
```

Comment:
It looks like we are missing configurations for the other transfers columns.  We should add explicit configurations rather than letting entity framework work them out.